Makefile

```
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+.ONESHELL:
+.SHELLFLAGS 	 := -eu -o pipefail -c
+SHELL 			 := bash
+MAKEFLAGS 		 += --warn-undefined-variables
+MAKEFLAGS 		 += --no-builtin-rules
+IMAGE_REPOSITORY := ghcr.io/gardener/ingress-gce
+IMAGE_TAG        := $(shell git describe --tags --always)
```

Could you please replace tab characters with spaces?

```suggestion
.SHELLFLAGS      := -eu -o pipefail -c
SHELL            := bash
MAKEFLAGS        += --warn-undefined-variables
MAKEFLAGS        += --no-builtin-rules
IMAGE_REPOSITORY := ghcr.io/gardener/ingress-gce
IMAGE_TAG        := $(shell git describe --tags --always)
```

---

done